File: 计算表达式.py
#https://www.tzcoder.cn/acmhome/problemdetail.do?method=showdetail&id=1482
from math import *
def f(x,n):
    if n==1:
        y=sqrt(n+x)
    else:
        y = sqrt(n+f(x,n-1))
    return y
print(f(0,3))

File: 计算表达式.cpp
#include<stdio.h>
#include<math.h>
double f(double x,int n)
{
    double y;
    if(n == 1)
    {
        y = sqrt(n + x);
    }
    else
        y = sqrt(n + f(x,n - 1));
    return y;
}
int main()
{
    double x;
    int n;
    double y;
    scanf("%lf%d",&x,&n);
    y = f(x,n);
    printf("%.2lf\n",y);
    return 0;
}
